### PR TITLE
created vehicle Model with converters

### PR DESCRIPTION
**Pull request by kevinhuynh269**
Created a vehicle model with converters please accept my request

### CODE REVIEW

File: src/app/models/vehicle.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+// Converts JSON strings to/from your types
+export class Convert {
+    public static toVehicle(json: string): Vehicle {
+        return JSON.parse(json);
+    }
+
+    public static vehicleToJson(value: Vehicle): string {
+        return JSON.stringify(value);
+    }
+}
```
Comment (psalaza) [inline src/app/models/vehicle.ts:81]: So clean.

**Review by psalaza — APPROVED** [review]

File: src/app/models/vehicle.ts (hunk 1 above)
Comment (RexBrown122) [reply src/app/models/vehicle.ts:81]: Looks good to me

**Comment by RexBrown122** [comment]
Looks good to me

**Comment by LWooler** [comment]
I approve this addition

**Pull request merged**